docs/spec/governance/state.md

```diff
@@ -2,47 +2,58 @@
 
 ## State
 
-### Procedures
+### Procedures and base types
 
 `Procedures` define the rule according to which votes are run. There can only 
 be one active procedure at any given time. If governance wants to change a 
 procedure, either to modify a value or add/remove a parameter, a new procedure 
 has to be created and the previous one rendered inactive.
 
+
 ```go
+type Procedure struct {
```

Comment:
I think we should split Procedure into a couple separate Procedures, for different parts of governance, such as Deposits, Voting, Tallying, etc.

```
type DepositProcedure struct {
    MinDeposit        sdk.Coins
    MaxDepositPeriod  int64
}

type TallyingProcedure struct {
    Threshold         rational.Rational
    Veto              rational.Rational
    GovernancePenalty sdk.Rat
}

type VotingProcedure struct {
    VotingPeriod      int64
}
```

Reply:
Why is that?

Reply:
ok